packages/dds/sequence/src/test/intervalCollection.undoRedo.spec.ts

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+
+import {
+	MockContainerRuntimeFactory,
+	MockFluidDataStoreRuntime,
+	MockStorage,
+} from "@fluidframework/test-runtime-utils";
+import {
+	appendToMergeTreeDeltaRevertibles,
+	MergeTreeDeltaRevertible,
+	revertMergeTreeDeltaRevertibles,
+} from "@fluidframework/merge-tree";
+import { SharedString } from "../sharedString";
+import { IntervalCollection, IntervalType, SequenceInterval } from "../intervalCollection";
+import { SharedStringFactory } from "../sequenceFactory";
+import { assertIntervals } from "./intervalUtils";
+
+describe("Undo/redo for interval collection operations", () => {
```

Comment:
Add a comment somewhere in this file that the code under test is currently doing the wrong thing so the tests are validating the bug and should be updated when fixed.